contrib/coreos/user-data.example

```diff
@@ -133,3 +134,14 @@ write_files:
       TOOLBOX_DOCKER_IMAGE=ubuntu-debootstrap
       TOOLBOX_DOCKER_TAG=14.04
       TOOLBOX_USER=root
+  - path: /etc/environment_proxy
```

Comment:
That's a possibility, too. We'd just need to write out COREOS_PRIVATE_IPV4 and COREOS_PUBLIC_IPV4. That may also remove the logic to source /etc/environment_proxy as well.

```
- path: /etc/environment
  content: |
    COREOS_PUBLIC_IPV4=$public_ipv4
    COREOS_PRIVATE_IPV4=$private_ipv4
    HTTP_PROXY=
    [...]
```